.github/workflows/main.yml

```diff
@@ -11,8 +11,11 @@ jobs:
         target: [test-clj, test-cljs]
 
     steps: 
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+
     - name: Setup CI Environment
-      uses: yetanalytics/actions/setup-env@v0
+      uses: yetanalytics/actions/setup-env@v0.0.4
 
-    - name: Run make target $${{ matrix.target }}
+    - name: Run make target ${{ matrix.target }}
```

Comment:
curious about this change, what was the extra `$` doing?

Reply:
I have the same question. My guess is that it was a typo and I overlooked it since I (incorrectly) assumed it was doing something special.

Reply:
Yeah, super weird that it just worked fine before, I can't find anything on it in the actions expression or workflow syntax docs. Oh well!

Reply:
It most certainly works now too.